config/cable.yml

```diff
@@ -0,0 +1,8 @@
+development:
+  adapter: async
+
+test:
+  adapter: async
+
+production:
+  adapter: async
```

Comment:
what should we use in production?

Reply:
on RoR guides they mention Redis, is it something we support? 
checkout: https://edgeguides.rubyonrails.org/action_cable_overview.html#subscription-adapter

```yml
production:
  adapter: redis
  url: redis://10.10.3.153:6381
  channel_prefix: appname_production
```

Reply:
Due to Dynflow there is a Redis instance present and configured for persistence so either that or PostgreSQL can be counted on I think. I don't know how the two compare.